Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -36,12 +36,38 @@ public abstract class ViewRenderer<TView, TNativeView> : VisualElementRenderer<T
 #endif
 		NativeColor _defaultColor;
 
+		event EventHandler<PropertyChangedEventArgs> _elementPropertyChanged;
+		event EventHandler _controlChanging;
+		event EventHandler _controlChanged;
+
+
+
 		protected virtual TNativeView CreateNativeControl()
 		{
 			return default(TNativeView);
 		}
 
 		public TNativeView Control { get; private set; }
+		NativeView IVisualNativeElementRenderer.Control => Control;
+
+
+		event EventHandler<PropertyChangedEventArgs> IVisualNativeElementRenderer.ElementPropertyChanged
+		{
+			add { _elementPropertyChanged += value; }
```

Comment:
maybe use WeakEventManager

Reply:
I think that might add too much overhead without enough gain.  

This is primarily used internally by the static managers which get unhooked from the dispose. Users could tie into this by casting the renderer to the interface but then it'd be on them to unwire it (which makes sense).  I think WeakEventManagers make more sense at the Core level where we are the ones hooking into events on user owned classes i.e. Commands and INCC on User controlled lists